office-js' is not in the npm registry

It does not seem to be possible to run this, since doing "npm install" results in:

```
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/office-js
npm ERR! 404
npm ERR! 404 'office-js' is not in the npm registry.
```

--- a/.gitignore
+++ b/.gitignore
@@ -18,3 +18,6 @@ node_modules/
 # temp files
 models/temp/
 
+
+models/appSettings.js
+bin/

--- a/package.json
+++ b/package.json
@@ -7,8 +7,8 @@
   },
   "description": "An Office 365 API app with Node, Express, Ejs and more.",
   "author": {
-    "name": "kdeding",
-    "email": "kdeding@microsoft.com"
+    "name": "Michael Mainer",
+    "email": "mmainer@microsoft.com"
   },
   "dependencies": {
     "body-parser": "~1.8.1",
@@ -21,7 +21,6 @@
     "express-session": "~1.0.0",
     "method-override": "~1.0.0",
     "morgan": "~1.3.0",
-    "office-js": "0.0.1",
     "passport": "~0.1.17",
     "passport-azure-oauth": "0.0.6",
     "request": "^2.51.0",

